ingeniamotion/communication.py

```diff
@@ -999,17 +1001,41 @@ def load_firmware_canopen(
                 error_enabled_callback,
             )
 
-    def load_firmware_ecat(self, ifname: str, fw_file: str, slave: int = 1) -> None:
+    @staticmethod
+    def __get_boot_in_app(fw_file: str) -> bool:
+        """Return true if the bootloader is included in the application (file extension is .sfu).
```

Comment:
This is not obvious, I would include this on the description

CoCo-MoCo products have boot_in_app, and exclusively use .sfu fw file formats.
CoMoCo products do not have boot in app, and exclusively use .lfu fw file formats